config/locales/en.yml

```diff
@@ -21,3 +21,6 @@
 
 en:
   hello: "Hello world"
+  success_create: 'Project was successfully created.'
```

Comment:
items related to projects are not separated. it prevents adding more locals with same keys.
also locals keys are incorrect
replace `success_create` with `successful_create`. this have more sense